src/swupd.h

```diff
@@ -409,6 +405,27 @@ enum swupd_code check_update();
 
 /****************************************************************/
 
+/* swupd_comp_funtions.c */
+
+int find_manifest(const void *a, const void *b);
+int find_filerecord(const void *a, const void *b);
+int find_file(const void *a, const void *b);
+int find_subscription(const void *a, const void *b);
+
+int cmp_file_filename(const void *a, const void *b);
+int cmp_file_filename_ptr(const void *a, const void *b);
+int cmp_file_hash(const void *a, const void *b);
+int cmp_filerecord_filename(const void *a, const void *b);
+int cmp_manifest_component(const void *a, const void *b);
+int cmp_subscription_component(const void *a, const void *b);
+
+int sort_fullfile(const void *a, const void *b);
```

Comment:
The name of this function is not good. The function doesn't sort anything, it's a compare function.
We could have a better naming standard for cmp functions.
Maybe something like: `cmp_ + type of first parameter + type of second parameter + some other sufix.`
So in this case `cmp_file_file`.

Reply:
The same applies to all find_, sort_, filter_ functions here